gui/system/forms.py

```diff
@@ -1129,11 +1129,11 @@ def middleware_clean(self, data):
         return data
 
     def done(self, request, events):
-        if self.original_instance['adv_consolemsg'] != self.instance.adv_consolemsg:
-            if self.instance.adv_consolemsg:
-                events.append("_msg_start()")
-            else:
-                events.append("_msg_stop()")
+        if self.instance.adv_consolemsg:
```

Comment:
I think this change should be backed out, or it will run _msg_start() even when consolemsg has not changed.

Reply:
Thanks change was necessary as it never really works. Original instance and the instance always matched so without this, changing the settng and saving had no effect without a reload. 

I saw no ill effects having it run msg_start multiple times.